app/views/budgets/investments/_milestones.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="tabs-panel tab-milestones" id="tab-milestones">
+  <div class="row">
+    <div class="small-12 column">
+      <% if @investment.milestones.blank? %>
+        <div class="callout primary text-center">
+          <%= t('budgets.investments.show.no_milestones') %>
+        </div>
+      <% end %>
+      <section class="timeline">
+        <ul class="no-bullet">
+          <% @investment.milestones.each do |milestone| %>
+            <li>
+              <div>
+                <h3><%= milestone.title %></h3>
+                <strong>Publicado el <%= milestone.created_at.strftime("%d/%m/%Y") %></strong>
```

Comment:
I think the "Publicado el" should go as part of a translation literal, right? You can interpolate like:

`t("milestones.publish_date", publish_date:  milestone.created_at.strftime("%d/%m/%Y"))`

with `publish_date: "Publicado el %{publish_date}` on the .yml file